cargo-miri-test/stdout.ref

```diff
@@ -0,0 +1 @@
+0x01020304
```

Comment:
the leading zero isn't in the printed output.

r=me with travis happy

Reply:
*oops* I fixed the format string, it should now be there.